pylti/tests/test_flask_app.py

```diff
@@ -1,38 +1,56 @@
+"""
+Test pylti/test_flask_app.py module
+"""
 from flask import Flask, session
 
-from pylti.flask import lti
+from pylti.flask import lti as lti_flask
 from pylti.common import LTI_SESSION_KEY
 
-
-app = Flask(__name__)
+app = Flask(__name__) # pylint: disable=invalid-name
 
 
 class ExceptionHandler(object):
+    """
+    Custom exception handler.
+    """
     exception = None
 
     def set(self, exception):
+        """
+        Setter: set class variable exception.
+        """
         self.exception = exception
 
     def get(self):
+        """
+        Return exception if not None otherwise returns None
+        """
         if self.exception is None:
             return None
         else:
             return self.exception['exception']
 
     def reset(self):
+        """
+        Reset variable exception
+        """
         self.exception = None
 
-app_exception = ExceptionHandler()
+app_exception = ExceptionHandler() # pylint: disable=invalid-name
 
 
 def error(exception):
+    """
+    Set exception to exception handler and returns error string.
+    """
     app_exception.set(exception)
     return "error"
 
 
 @app.route("/unknown_protection")
-@lti(error=error, app=app, request='notreal')
+@lti_flask(error=error, app=app, request='notreal')
```

Comment:
@carsongee because the name of decorator and the param was same. I was getting a warning from pylint which state that I am modifying outer variable. On changing the param of function i got error from test_flash.py test start getting fail